geo-replication/src/gsyncd.c

```diff
@@ -10,11 +10,11 @@
 #include <glusterfs/compat.h>
 #include <glusterfs/syscall.h>
 
-#include <stdlib.h>
 #include <stdio.h>
-#include <unistd.h>
+#include <stdlib.h>
```

Comment:
Any reason for changing the order of the includes ?

Reply:
I suppose when we run clang-format on the file it rearranges it accordingly. That's all

Reply:
Current `.clang-format` configuration has `SortIncludes:    false`, so it shouldn't have happened. Can you try resubmit the patch without the changes in the include files ? it's better to keep changes as small as possible.

Reply:
I have made those changes and pushed it again and this time the clang format test is failing because of the rearranged includes

Reply:
I don't understand why it happens. Anyway, sorry for the noise. If you can resubmit your original patch I'll accept it.

Reply:
Is it because of change in clang version ? ( was mentioned in the community meeting ) I think a new clang version was integrated with the CI and maybe that is causing it to fail.

Reply:
> I don't understand why it happens. Anyway, sorry for the noise. If you can resubmit your original patch I'll accept it.

Thank you. I have resubmitted the patch.